iocore/eventsystem/I_EThread.h

```diff
@@ -56,7 +56,7 @@ enum ThreadType {
   DEDICATED,
 };
 
-extern volatile bool shutdown_event_system;
+extern bool shutdown_event_system;
```

Comment:
That's part of the revert that was done on master by @zwoop. Perhaps we should address that there and then maybe add any changes that come out of that to this PR?